chapters/infrastructure/digital-rewards/reading/README.md

```diff
@@ -3,10 +3,38 @@
 Owner: Răzvan Nițu
 Reviewer: Mihai Băruța
 
-definitions, types
+Some teaching paradigms involve the use of a reward system to incentivize learners to study.
+To enable the use of such teaching techniques, we offer an infrastructure for the use of cryptocurrency rewards that may be used to stimulate both learning and contributions to the educational content.
+We do not discuss the efectiveness of reward-based learning techniques, we simply provide the tools to implement such a system if an educator chooses to use it.
```

Comment:
But shouldn't we discuss it? Why would someone use cryptocurrency if we don't make a case for it?